ansible/install.yml

```diff
@@ -17,17 +17,8 @@
        - iptables
        - software-properties-common
 
-   - name: install Docker CE repos (1/3)
-     shell: curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-
-   - name: install Docker CE repos (2/3)
-     shell: add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
-
-   - name: install Docker CE repos (3/3)
-     shell: apt-get -qq update
-
    - name: install Docker CE
-     apt: name=docker-ce state=installed
```

Comment:
Hi @prashplus ... interesting. Why should we change this?

The official Docker documentation still describes the existing installation approach:
https://docs.docker.com/install/linux/docker-ce/ubuntu/

Can you describe the rational behind your changes?

Reply:
Its very quick and easy way to setup docker. And the script at https://get.docker.com is very generic.

Reply:
True. But why should we use a different approach than the officially documented one for ubuntu ? since the ansible install script is ubuntu specific anyhow.

Reply:
Its also using apt repos if you look at the script. It's just for reducing LOC and code complexity.

Reply:
But still, it downloads unverified shell scripts and executes them ... not a good practice to have this in a Ansible script just to safe 2-3 LOC.

If you have a look at the script's code:

> NOTE: Make sure to verify the contents of the script
>       you downloaded matches the contents of install.sh
>       located at https://github.com/docker/docker-install
>       before executing.

Who would check this before running the Ansible script? That is the point.

Reply:
Ok fine...